packages/utilities/src/diff/doc/IWebDiffManager.ts

```diff
@@ -0,0 +1,24 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+/**
+ * Web help manager API that handles launching of web help and generating it if necessary.
+ * @export
+ * @interface IWebDiffManager
+ */
+export interface IWebDiffManager {
+    /**
+     * launch the diff of two file in web
+     * @param {string} content Html content string to launch in web
+     */
+    openDiffs(content: string): void;
+
+}
```

Comment:
(same comment as above)

Creating the interface is only necessary if we use it elsewhere. Currently this one seems unused unless I have missed where we use it.

If the interface is desired, I would suggest updating the WebDiffManager class to implement it (`export class WebDiffManager implements IWebDiffManager`). Or if it is not needed anywhere, it may be ok to remove it?

Reply:
Yes, this interface is already implemented in WebDiffManager.ts.